modules/nf-core/glimpse2/phase/main.nf

```diff
@@ -69,4 +69,23 @@ process GLIMPSE2_PHASE {
         glimpse2: "\$(GLIMPSE2_phase --help | sed -nr '/Version/p' | grep -o -E '([0-9]+.){1,2}[0-9]' | head -1)"
     END_VERSIONS
     """
+
+    stub:
+    def region = input_region    ? "${output_region.replace(":","_")}" : "${reference}"
+    def args   = task.ext.args   ?: ""
+    def prefix = task.ext.prefix ?: "${meta.id}_${region}"
+    def suffix = task.ext.suffix ?: "bcf"
+    """
+    if $input_bam
+    then
+        touch all_bam.txt
```

Comment:
Is this needed? I don't see it declared as an `output`

Reply:
You only need to generate the files that are declared as `output` as those are the only staged by the process and hence will be the only validated by any test or expected during the `stub` run. I mean, it doesn't harm that they are generated but just not necessary.

Reply:
Okay !